ControlSettings/ControlSettings.cpp

```diff
@@ -248,16 +248,29 @@ namespace Plugin {
                , m_isDisabledHdmiIn4KZoom (false)
     {
         LOGINFO("Entry\n");
-        devicePtr = new WPEFramework::Plugin::TV;
+#ifndef ENABLE_TV_SUPPORT
```

Comment:
in recipe file define something like  #define CONFIG_DEVICE_TYPE TV or STB
and use that variable here to create object of TV or STB class

Reply:
Thank you, Amit. "CONFIG DEVICE TYPE TV or CONFIG DEVICE TYPE STB" will be the new name.
Will change it accordingly.